tests/testthat/test-run_stress_test.R

```diff
@@ -1,23 +1,26 @@
-test_that("output includes argument names with suffix '_arg'", {
-  is_me <- identical(path.expand("~"), "/home/mauro")
-  skip_if_not(is_me)
+test_that("with bonds output is unchanged and names have the suffix '_arg'", {
+  skip_on_ci()
+  skip_on_cran()
 
-  in_agnostic <- "/home/mauro/tmp/st/ST_INPUTS_MASTER"
-  in_specific <- "/home/mauro/tmp/st/ST_TESTING_BONDS/inputs"
+  in_agnostic <- Sys.getenv("ST_AGNOSTIC")
+  in_specific <- Sys.getenv("ST_SPECIFIC_BONDS")
   out <- tempfile()
   fs::dir_create(out)
 
-  x <- suppressWarnings(capture.output(
-    run_stress_test("bonds",
+  suppressed_console_output <- suppressWarnings(capture.output(
+    results <- run_stress_test("bonds",
       input_path_project_agnostic = in_agnostic,
       input_path_project_specific = in_specific,
       output_path = out,
-      term = 1:2
+      term = 1:2,
+      return_results = TRUE
     )
   ))
 
+  expect_snapshot(lapply(results, as.data.frame))
+
   csv_files <- fs::dir_ls(out, regexp = "[.]csv$")
```

Comment:
my monkey brain does not fully understand where the csv files are read from here. Since you call run_stress_test with return_results = TRUE results are not written to the output path, so how can there be csv files in out? It only holds the logfile if I try running it interactively?

Reply:
Thanks! This was a bug in the test. The result changed but the expectation remained the same. It's now fixed.